frontend/packages/console-shared/src/selectors/common.ts

```diff
@@ -32,3 +32,7 @@ export const getAnnotations = <A extends K8sResourceCommon = K8sResourceCommon>(
   value: A,
   defaultValue?: K8sResourceCommon['metadata']['annotations'],
 ) => (_.has(value, 'metadata.annotations') ? value.metadata.annotations : defaultValue);
+export const getTaints = <A extends K8sResourceKind = K8sResourceKind>(
```

Comment:
@rawagner can you please approve this addition? :)

Reply:
A better type would be `K8sResourceCommon` which is more generic than `K8sResourceKind`

Reply:
@rawagner 
`K8sResourceCommon` doesn't have the path `spec.*` which the selector needs, it will throw an error.
`K8sResourceKind` has it.